### ISSUE TITLE
Miner flooding network and disabling network access

### ISSUE THREAD

**Issue by Roaders** [nanopool/Claymore-Dual-Miner#181]
Hi

I am getting some very odd behaviour with my rigs. More details [here](https://superuser.com/questions/1283877/one-machine-on-network-switch-seems-to-be-killing-network-for-others-hacked?noredirect=1#comment1898423_1283877) but I will summarise.

I have 3 rigs. One has 10 cards, one currently has 2 cards and the other just has 1 card. All cards are GTX 1070s.

Each rig is Ubuntu server 16.x

I run one instance of claymore for each card and this works fine for hours. I have previously had a rig with 12 cards running like this for days. This means that I will have 10 - 20 instances of Claymore.

At the moment (and only after adding a second rig I think) the rigs will run for some variable amount of time - say 9 hours and then they will all stop responding to pings or ssh.

This seems to be a network issue with one of the rigs (the rig that swamps the network isn't always the same one). If I disconnect the machine from the network the other 2 start responding again and are fine. If I plug the affected computer back in again the network goes down again.

To recover the lost machine I have to reboot it.

Has anyone ever seen anything like this when running Claymore on multiple rigs? It's a bit of a long shot but I thought I would post here to see if anyone had ever seen anything similar.

**Comment by jwb6658** [comment]
I’ve seen this when there has been ip overlap. Do you use static addressing?

**Comment by Roaders** [comment]
No, I use DHCP and Google Wifi has reserved IPs for all the machines.
It also just randomly happens when all 3 machines have been running for some time - it's not like it happens when I turn one on or something.

It's been driving me mad!! I've spent so long trying to debug suspected hardware faults ect! I really need to get this sorted.

**Comment by jwb6658** [comment]
I’d recommend using statically assigned ip’s and working through step by step troubleshooting of one machine at a time. 

Thanks,
Jarrod
